### PR TITLE
Update txt2tags.py

### PR DESCRIPTION
**Pull request by goetzschnell**
added target ConTeXt (ctx),
topics to discuss eventually:
- escaping
- some rules
- output (layout and design of ConTeXt output)

### CODE REVIEW

File: txt2tags.py [hunk 1]
```diff
@@ -82,7 +82,7 @@
 my_url = "https://txt2tags.org"
 my_name = "txt2tags"
 my_email = "jendrikseipp@gmail.com"
-__version__ = "3.7"
+__version__ = "3.7.1"
```
Comment (jendrikseipp) [inline txt2tags.py]: No need to change the version.

File: txt2tags.py [hunk 2]
```diff
@@ -404,7 +405,59 @@
 %(HEADER1)s
 %(HEADER2)s
 %(HEADER3)s
-"""
+""",
+   "ctx": \
+r"""%% created from txt2tags
```
Comment (jendrikseipp) [inline txt2tags.py]: No need for the comment.

File: txt2tags.py (hunk 1 above)
Comment (goetzschnell) [reply txt2tags.py]: eliminated

File: txt2tags.py (hunk 2 above)
Comment (goetzschnell) [reply txt2tags.py]: eliminated

**Comment by goetzschnell** [comment]
eliminated version and comment,
added samples/sample.ctx and samples/sample.pdf
ran run.py, added run.pyc in every test folder
result of run.py:
[2020-02-12result.txt](https://github.com/txt2tags/txt2tags/files/4194790/2020-02-12result.txt)

**Comment by goetzschnell** [comment]
eliminated sample.pdf
eliminated *.pyc

**Comment by jendrikseipp** [comment]
Merged. Thanks!

**Pull request merged**